app/components/list-item/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { ListItemType } from '@/app/app-types'
+
+type ListItemProps = {
+  items: ListItemType[]
+}
+
+const ListItem = (props: ListItemProps) => {
+  return (
+    <ul data-testid="list-item-wrapper" className="pl-4">
+      {props.items.map((listItem) => {
+        return (
+          <li key={listItem.id} className="flex gap-4 items-center">
```

Comment:
Add a null check on listItem and render li only if it is not null

Reply:
Added the null check before rendering 'listItem'.